neuralop/data/transforms/transforms.py

```diff
@@ -73,6 +64,79 @@ def to(self, device):
         self.transforms = [t.to(device) for t in self.transforms if hasattr(t, 'to')]
         return self
 
+class DictTransform(Transform):
```

Comment:
Why is this duplicated? We already have it in base_transforms no?

Reply:
Yeah it was, just removed this file